go.mod

```diff
@@ -510,7 +510,7 @@ replace (
 	k8s.io/klog/v2 => k8s.io/klog/v2 v2.4.0
 	k8s.io/kube-aggregator => ./staging/src/k8s.io/kube-aggregator
 	k8s.io/kube-controller-manager => ./staging/src/k8s.io/kube-controller-manager
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20201113171705-d219536bb9fd
+	k8s.io/kube-openapi => github.com/liggitt/kube-openapi v0.0.0-20210113222446-36865573704e
```

Comment:
what's the story here @liggitt ? (temporary redirect to personal repo)?

Reply:
just testing until https://github.com/kubernetes/kube-openapi/pull/222 merges, then will replace the WIP commit with a real one

Reply:
done